### ISSUE TITLE
Problem export SCORM 1.2

### ISSUE THREAD

**Issue by GoogleCodeExporter** [PawarPawan/xerteonlinetoolkits#18]
```
What steps will reproduce the problem?
1. Export SCORM 1.2
2. Attempt to import into LMS

What is the expected output? 
Successful import

What do you see instead?
Just a fail, attempt again message.

What version of the product are you using? On what operating system?
Local installation of XOT 1.9 on Windows 7

Please provide any additional information below.

I can export an archive zip file, import it into the JISC TechDis Xerte sandpit 
and then export it as a SCORM 1.2 file and the import into my LMS works just 
fine.

So, to summarize, SCORM 1.2 export from local install fails and the SCORM 1.2 
export from the sandpit of the same source succeeds.

Any ideas?

```

Original issue reported on code.google.com by `NBAi...@gmail.com` on 2 Feb 2013 at 11:57
